hack/linter.sh

```diff
@@ -18,9 +18,16 @@ set -e -o pipefail
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+VERSION=1.23.1
+
+if [[ $(golangci-lint --version | grep $VERSION | wc -l) -eq 0 ]]; then
```

Comment:
This fails when `golangci-lint` is not yet installed

Reply:
🤦‍♂